book/website/index.md

```diff
@@ -1,75 +1,124 @@
+Our moonshot goal is to make reproducibility "too easy not to do"! 
+Whether you are a researcher, coder, project leader, community connector, funder, policymaker, or just curious about data, your unique role is crucial in this journey.
 
-In the {ref}`Foreword Section<fw>` of this book, we provide meta-level information about _The Turing Way_, including guidance on how to use and navigate this project, how to cite the book and opportunities to get involved.
+Let's make data science inclusive, together! 
+Visit our [start page](https://the-turing-way.start.page/) to get started. 🚀
 
 ```{figure} figures/theturingway-pathway.*
 ---
 name: welcome-image
 alt: The Turing Way project is illustrated as a road or path with shops for different data science skills. People can go in and out with their shopping cart and pick and choose what they need.
 ---
-_The Turing Way_ project illustration by Scriberia. Zenodo. [http://doi.org/10.5281/zenodo.3332807](http://doi.org/10.5281/zenodo.3332807)
+_The Turing Way_ guides - an illustration by Scriberia and _The Turing Way_ community shared under the CC-BY 4.0 license on Zenodo. [DOI: 10.5281/zenodo.3332807](http://doi.org/10.5281/zenodo.3332807)
 ```
 
-The book is collaboratively written and built in the open from the start.
-You can explore different opportunities to get involved on the [start page](https://the-turing-way.start.page/).
-Join [our Slack Workspace](https://join.slack.com/t/theturingway/shared_invite/zt-fn608gvb-h_ZSpoA29cCdUwR~TIqpBw) to connect and discuss your ideas and [subscribe to our newsletter](https://buttondown.email/turingway) to follow updates from our community.
-
 ## Different Pathways
 
+
 (welcome-community)=
-## Our Community
+## Join _The Turing Way_
 
-_The Turing Way_ community is dedicated to making collaborative, reusable and transparent research "too easy not to do".
-That means investing in the socio-technical skills required to work in a team, to build something more significant than any individual could deliver alone.
+Our goal is to provide individuals and organizations in academia, industry, government, and the public sectors the information they need to ensure that their projects are easy to reproduce and reuse. 
+We encourage all stakeholders in data science to utilise _The Turing Way_ to understand their roles and responsibilities for reproducibility.
 
-_The Turing Way_ is:
+**What Awaits You**
 
-* a book
-* a community
-* a global collaboration
+* Collaboration Beyond Boundaries: Engage in a global collaboration that transcends borders and welcomes diverse perspectives.
+* Skill Development: Learn a skills, share a skill that you have and contribute to projects in making a real impact.
```

Comment:
```suggestion
* Skill Development: Learn or share a skill that you have and contribute to projects in making a real impact.
```
There's a typo here but I'm not sure what it should be corrected to.

Reply:
I would like to go back to the original content: https://book.the-turing-way.org/#our-community - it's simple! What do you think?